galley/pkg/server/server.go

```diff
@@ -199,13 +203,21 @@ func (s *Server) Close() error {
 }
 
 //RunServer start Galley Server mode
-func RunServer(sa *Args, printf, fatalf shared.FormatFn) error {
+func RunServer(sa *Args, printf, fatalf shared.FormatFn, livenessProbeController *probe.Controller,
+	readinessProbeController *probe.Controller) error {
```

Comment:
`probe.Controller` is an interface and shouldn't be passed a pointer. This could be collapsed since liveness and readienss controller share the same type.

```golang
func RunServer(sa *Args, printf, fatalf shared.FormatFn, livenessProbeController, readinessProbeController *probe.Controller) error {
```

And the defered function could be called directly as well (see previous comment)

Reply:
done